pnpm/test/verifyDepsBeforeRun/multiProjectWorkspace.ts

```diff
@@ -548,83 +544,6 @@ test('multiple lockfiles', async () => {
   }
 })
 
-test('filtered install', async () => {
```

Comment:
I guess you should keep the filtered install test. Just change the expected result.

Reply:
https://github.com/pnpm/pnpm/pull/8645/commits/131681713a1d91108e24d30dd2b3edd755967525